pro/src/components/pages/Offers/Offer/Stocks/ThingStocks.jsx

```diff
@@ -59,16 +65,25 @@ const ThingStocks = ({ offer, reloadOffer }) => {
     : `/offre/${offer.id}/individuel/recapitulatif`
   const offersSearchFilters = useSelector(searchFiltersSelector)
   const offersPageNumber = useSelector(searchPageNumberSelector)
+  const mountedRef = useRef(true)
+
+  useEffect(() => {
+    return () => {
+      mountedRef.current = false
```

Comment:
pourquoi on a besoin d'ajouter cette ref ?

Reply:
pour enlever ce genre de warning :

```
Warning: Can't perform a React state update on an unmounted component.
This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in a useEffect cleanup function.
```

si t'as mieux je veux bien !

Reply:
Je crois qu'il vaudrait mieux régler le problème sous-jacent, peut-être pas dans cette PR (on peut préparer un ticket pour les ité de dette), et en attendant laisser le warning. 
Il y a tellement de variables qui dépendent les unes des autres dans les tableaux de deps qu'on ne contrôle pas l'appel des fonctions, il faudrait qu'on cleane ça pour faire en sorte que les fonctions ne soient appelées que quand on en a besoin.

Reply:
je les ai retiré/remis les warnings du coup